packages/core/src/generic/sql-fragment.ts

```diff
@@ -1,4 +1,4 @@
-import type { Falsy } from './falsy';
+import type { Falsy } from '@sequelize/utils';
```

Comment:
and here Falsy is too broad, it should be replaced with Nullish

Reply:
Replaced now.